CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,219 @@
+# Quansight Repository Code of Conduct
+
+1. Introduction
+
+    a. Expected Behavior
+    
+    b. Unacceptable Behavior
+2. Incident Reporting Guidelines
+
+    a. How to Submit an Incident Report
+    
+    b. Conflicts of Interest
+    
+    c. What to Include in a Report
+    
+    d. Procedures after Filing a Report
+    
+3. License
+
+-------
+
+## 1. Introduction
+
+The aim of this Code of Conduct is to empower all participants at Quansight-organized in-person events in Quansight-managed public repositories to facilitate healthy and constructive community behavior. Those participants may be users, maintainers, or contributors to open-source projects managed by Quansight LLC or Quansight Labs. This Code of Conduct outlines expected behaviors as well as steps for reporting incidents of unacceptable behavior.
+
+This Code of Conduct applies to all repositories and communities for Quansight-managed open-source projects regardless of whether or not the repository explicitly includes this document. The Code also applies in public spaces when an individual represents a project or its community. Examples include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or in-person event. What constitutes representing a project may be further defined and clarified by project maintainers.
+
+Quansight is committed to providing a welcoming and inspiring community for all. Anyone claiming affiliation with Quansight public repositories is required to conform to the Code of Conduct. This Code of Conduct applies to all spaces and public repositories of Quansight on all platforms (including Github). Possible consequences for breaching this Code of Conduct are listed below.
+
+For any queries regarding this Code of Conduct, please contact any member of the Code of Conduct committee. The Code of Conduct committee (for 2021) is:
```

Comment:
I saw Ralf had a comment about the "any member" wording here, but I didn't see if it was addressed (the wording is still here).

Reply:
Ralf's comment was specifically regarding submitting a formal report. (i.e., later in the document) to all members so that doesn't really apply here.

The line above is about seeking information, i.e., if someone wants to ask questions about interpreting the CoC. The intent here is to make it easy to ask questions early before a small disagreement becomes a big one (indeed, to encourage people to do so).

I'll reword it so that intention is more explicit.